drizzlepac/runastrodriz.py

```diff
@@ -270,10 +304,10 @@ def process(inFile, force=False, newpath=None, num_cores=None, inmemory=True,
         # Make sure given filename is complete and exists...
         inFilename = fileutil.buildRootname(inFile, ext=['.fits'])
         if not os.path.exists(inFilename):
-            print("ERROR: Input file - %s - does not exist." % inFilename)
+            log.error(f"ERROR: Input file - {inFilename} - does not exist.")
```

Comment:
The "ERROR" string is redundant with your formatting here and in a few other places below.


```suggestion
            log.error(f"Input file - {inFilename} - does not exist.")
```